doc/release-notes/release-notes-0.3.0.md

```diff
@@ -0,0 +1,23 @@
+Tapyrus Signer version 0.3.0 is now available for download at: 
+  https://github.com/chaintope/tapyrus-signer/release/tag/v0.3.0/
+    
+Please report bugs using the issue tracker at github:
+  https://github.com/chaintope/tapyrus-signer/issues
+  
+Project source code is hosted at github; you can get
+source-only tarballs/zipballs directly from there:
+  https://github.com/chaintope/tapyrus-signer/tarball/v0.3.0  # .tar.gz
+
+0.3.0 change log
+-------------------
+
+*Block Signing*
+
+* Upgraded block proof as fixed size Schnorr signature based threshold signature.
+
+*Node Starting Process*
+
+* Support reading options from toml format file.
```

Comment:
It is prefer to have a sample toml file and a description of how to specify it.

Reply:
I added!